tests/test_iiif.py

```diff
@@ -183,12 +181,51 @@ def test_get_image_when_image_server_gives_ConnectTimeout(
             == RESPONSE_CONTENT_ERROR_RESPONSE_FROM_IMAGE_SERVER + " ConnectTimeout"
         )
 
+    @patch("iiif.image_server.get_image_from_server")
+    @patch("iiif.metadata.do_metadata_request")
+    def test_get_info_json(self, mock_do_metadata_request, mock_get_image_from_server, client):
+        mock_do_metadata_request.return_value = MockResponse(
```

Comment:
is 1 test voldoende voor de info json? kan hoor, vind ik zelf lastiger in te schatten

Reply:
Ja in principe wel. De combinatie met al die andere varianten (onbereikbare image of metadata servers, misformatted urls, etc) zijn allemaal dezelfde call, en worden dus door de andere tests afgevangen.